### PR TITLE
[MOSIP-31790]  "Deployment of external services" bugs fixed …

### PR DESCRIPTION
**Pull request by Mohanraj209**
…for platform upgrade.

### CODE REVIEW

File: deployment/v3/external/object-store/minio/README.md [hunk 1]
```diff
@@ -34,10 +34,19 @@ To install the MinIO client, follow the procedure from [here](https://docs.min.i
...
+### Note:
+* Make sure to replace <src-alias-name>, http://<server-name>:9000, <minio-root-user> and <minio-root-password> with your original values before exicuting the below commands.
+* --api S3v2: This specifies the API version being used. In this case, it's specifying version 2 of the S3 API, which is used for interacting with object storage systems like MinIO.
 * Set `alias` for MinIO servers.
+
+
   ```
     mc alias set <src-alias-name> http://<server>:9001 <minio-root-user> <minio-root-password> --api S3v4  ( old version )
-    mc alias set <dest-alias-name> http://<server>:9100 <minio-root-user> <minio-root-password> --api S3v4  ( latest versions )
+    mc alias set <dest-alias-name> https://<server>:9100 <minio-root-user> <minio-root-password> --api S3v4  ( latest versions )
```
Comment (gsasikumar) [inline deployment/v3/external/object-store/minio/README.md:49]: Is there a way to get the http vs https from over all system deployment.

File: deployment/v3/external/postgres/generate-secret.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
   namespace: postgres
 type: Opaque
 data:
-  postgresql-password: {postgres_base64_password}
+  postgres-password: {postgres_base64_password}
```
Comment (gsasikumar) [inline deployment/v3/external/postgres/generate-secret.py:41]: is it encrypted base64 ?

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: deployment/v3/external/postgres/generate-secret.py (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/external/postgres/generate-secret.py:41]: It is Base64 encoding where the password changes into base64 format and is stored as a secret, it isn't a form of encryption.

File: deployment/v3/external/object-store/minio/README.md (hunk 1 above)
Comment (ckm007) [reply deployment/v3/external/object-store/minio/README.md:49]: let's replace these url's with <source minio URL> and <destination minio URL>.

Comment (Mohanraj209) [reply deployment/v3/external/object-store/minio/README.md]: updated.

**Review by ckm007 — APPROVED** [review]

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**